requirements.txt

```diff
@@ -1,7 +1,8 @@
 aiofiles==0.3.1
 bundle-placement==0.0.1
 Jinja2==2.9.6
-juju==0.5.3
+# juju==0.5.3
+git+https://github.com/juju/python-libjuju@17b26ef
```

Comment:
@johnsca We should probably get a release out of libjuju, otherwise this is pinned at the commit with the set/get_config methods.